### ISSUE TITLE
Find null values

### ISSUE THREAD

**Issue by omgMath** [jclo/picodb#8]
Hi :)
I think I found a bug - unless this intentional.

Reproduction:
```JavaScript
var PicoDB = require('picodb');
var db = PicoDB();
db.insertMany([{a: 1, b: null}, {a: 2, b: 2}], function (err, docs) {
    // docs contains the inserted documents with their unique id.
});
db.find({b: null}).toArray(function (err, docs) {
    console.log(docs.length);
});
```

Expected result:
`1`

Actual result:
`0`

Taking `MongoDB` as a reference, I get `1` as a result. Same happens with `{b: {$eq: null}}`.

**Comment by jclo** [comment]
You are right, it's a bug! It is fixed in the version 0.12.2 I am going to release.

**Comment by jclo** [comment]
My commit didn't close the issue! So, I close it manually.
